radis/lbl/broadening.py

```diff
@@ -1030,8 +1049,13 @@ def _add_doppler_broadening_HWHM(self, df, Tgas):
             - ``hwhm_gauss``
         """
 
+        print(df.head())
+        print(df.attrs)
+        molar_mass = get_molar_mass(df)
+
         # Calculate broadening HWHM
-        wg = doppler_broadening_HWHM(df.wav, df.molar_mass, Tgas)
+        print("it is here")
```

Comment:
BTW, I insert a lot of `raise` or stop points in my own debugging, then use `%debug` from an IPython console to get access to local variables, and go up/down the parent and child functions with `u` and `d`

Reply:
Thank you. Will try using this henceforth.